scripts/pi-hole/js/header.js

```diff
@@ -1,9 +0,0 @@
-/* Pi-hole: A black hole for Internet advertisements
-*  (c) 2017 Pi-hole, LLC (https://pi-hole.net)
-*  Network-wide ad blocking via your own hardware.
-*
-*  This file is copyright under the latest version of the EUPL.
-*  Please see LICENSE file for your rights under this license. */
-// Remove JS warning
-var jswarn = document.getElementById("js-warn-exit");
-jswarn.parentNode.removeChild(jswarn);
```

Comment:
Presumably this is not needed due to the addition of the `<noscript>` tag in `header.php`?

Reply:
Yup. Also its CSS loads only when JS is not available.

Reply:
if you use FF, just go to `about: config` and disable `javascript.enabled`. Then refresh to see the warning. And when done flip it back to true

Reply:
Confirmed it works as expected!